pages/Parachute.md

```diff
@@ -0,0 +1,8 @@
+The Parachute is a useful [Technical Gadget](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Technical-Gadget) which allows you to Glide through the World.
```

Comment:
Maybe clarify that it slows down your fall and prevents you from taking fall damage while gliding.